src/time-range-picker/combobox-picker/combobox-picker.tsx

```diff
@@ -285,8 +295,6 @@ export const ComboboxPicker = ({
     };
 
     const renderDropdown = () => {
-        if (!dropdownOpen) return;
```

Comment:
this caused an error in the unit tests

```
 Error: Uncaught [TypeError: relatedTarget.hasAttribute is not a function]
```

likely the dropdown was not mounted in time when Floating UI tried to access it